ni_measurementlink_service/_internal/parameter/_message.py

```diff
@@ -0,0 +1,113 @@
+import struct
+from typing import Any, Dict
+
+from google.protobuf.internal import encoder, wire_format
+from google.protobuf.message import Message
+
+from ni_measurementlink_service._internal.parameter._serializer_types import (
+    Decoder,
+    Encoder,
+    Key,
+    NewDefault,
+)
+
+
+def _message_encoder_constructor(field_index: int, is_repeated: bool, is_packed: bool) -> Encoder:
```

Comment:
This doesn't return an `Encoder` because `Encoder`'s 2nd parameter is `bytes` not `Message`
```suggestion
def _message_encoder_constructor(field_index: int, is_repeated: bool, is_packed: bool) -> Callable[[WriteFunction, Message, bool], int]:
```